README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
 The initial list of these values were created at our first company retreat, when Bonusly was a team of seven people.
 
 * [Go get results](https://github.com/bonusly/un-handbook#go-get-results)
-* [Communicate](https://github.com/bonusly/un-handbook#communicate)
+* [Be an excellent teammate](https://github.com/bonusly/un-handbook#be-an-excellent-teammate)
 * [Default to transparency](https://github.com/bonusly/un-handbook#default-to-transparency)
-* [Work smarter, not harder](https://github.com/bonusly/un-handbook#work-smarter-not-harder)
-* [Help out](https://github.com/bonusly/un-handbook#help-out)
+* [Work smarter](https://github.com/bonusly/un-handbook#work-smarter)
 * [Seek and embrace diversity](https://github.com/bonusly/un-handbook#seek-and-embrace-diversity)
 * [Delight the customer](https://github.com/bonusly/un-handbook#delight-the-customer)
 
 ### Go get results
```

Comment:
Add a line before the bullets that says something like, "This means we value behaviors that:"